.github/workflows/release.yml

```diff
@@ -51,9 +51,11 @@ jobs:
         uses: gradle/wrapper-validation-action@v1
       - name: sha256
         run: |
-          make buildplugin
           tag=$(git describe --tags --abbrev=0 --exact-match)
           release_version="${tag:1}"
+          echo "release_version=${release_version}" >> "$GITHUB_ENV"
+          export ORG_GRADLE_PROJECT_releaseVersion="${release_version}"
```

Comment:
We need to set this prior to running the build to ensure the artifact has the right version number.